sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala

```diff
@@ -301,8 +301,14 @@ object FileFormatWriter extends Logging {
     try {
       Utils.tryWithSafeFinallyAndFailureCallbacks(block = {
         // Execute the task to write rows out and commit the task.
-        dataWriter.writeWithIterator(iterator)
-        dataWriter.commit()
+        val taskAttemptID = taskAttemptContext.getTaskAttemptID
+        val (res, timeCost) = Utils.timeTakenMs {
+          logDebug("$taskAttemptID starts to write and commit.")
+          dataWriter.writeWithIterator(iterator)
+          dataWriter.commit()
+        }
+        logInfo(s"$taskAttemptID finished to write and commit. Elapsed time: $timeCost ms.")
```

Comment:
After some more thought, I think it's better to use SQL metrics for it. It's very hard to know max/min/avg by reading the logs.

@AngersZhuuuu I think you tried it before. Can you restore the work?

Reply:
> After some more thought, I think it's better to use SQL metrics for it. It's very hard to know max/min/avg by reading the logs.
> 
> @AngersZhuuuu I think you tried it before. Can you restore the work?

Yea, working on this